### PR TITLE
Fix install.sh for DS 2.0 and add comment to install_config.conf

### PR DESCRIPTION
**Pull request by zhongjiajie**
Make our install work for dolphinscheduler 2.0

### CODE REVIEW

File: dolphinscheduler-server/src/main/resources/config/install_config.conf [hunk 1]
```diff
@@ -34,17 +37,13 @@ password="xx"
...
-zkQuorum="192.168.xx.xx:2181,192.168.xx.xx:2181,192.168.xx.xx:2181"
-
 # zk root directory
 zkRoot="/dolphinscheduler"
 
 # registry config
 # registry plugin dir
 # Note: find and load the Registry Plugin Jar from this dir.
-registryPluginDir="/data1_1T/dolphinscheduler/lib/plugin/registry"
+registryPluginDir="lib/plugin/registry/zookeeper"
```
Comment (zhongjiajie) [inline dolphinscheduler-server/src/main/resources/config/install_config.conf:99]: @CalvinKirs I found out I have to set parameter `registryPluginDir` to `lib/plugin/registry/zookeeper` to make registry server load jar and work.

So what the parameter `registryPluginName` use for? Should I set `registryPluginDir` to `lib/plugin/registry` and `registryPluginName` to `zookeeper` and combine them to string `lib/plugin/registry/zookeeper` in my bash script?

**Review by lenboo — DISMISSED** [review]
+1

File: dolphinscheduler-server/src/main/resources/config/install_config.conf (hunk 1 above)
Comment (CalvinKirs) [reply dolphinscheduler-server/src/main/resources/config/install_config.conf:99]: Just to verify consistency

Comment (zhongjiajie) [reply dolphinscheduler-server/src/main/resources/config/install_config.conf:99]: Ok I got it and thanks.

File: dolphinscheduler-server/src/main/resources/config/install_config.conf [hunk 2]
```diff
@@ -15,54 +15,109 @@
...
+# **DO NOT** set this configuration same as the current path (pwd)
+installPath="/data1_1T/dolphinscheduler"
+
+# The user to deploy DolphinScheduler for all machine we config above. For now user must create by yourself before run `install.sh`
+# script. The user needs to have sudo privileges and permissions to operate hdfs. If hdfs is enabled than the root directory needs
+# to be created by this user
+deployUser="dolphinscheduler"
+
+# The directory to store local data for all machine we config above. Make sure user `deployUser` have permissions to read and write this directory.
+dataBasedirPath="/tmp/dolphinscheduler"
```
Comment (zhongjiajie) [inline dolphinscheduler-server/src/main/resources/config/install_config.conf:61]: For reviewer, I am not sure whether parameter `dataBasedirPath` belong to section `INSTALL MACHINE` or not

**Review by zhongjiajie — COMMENTED** [review]
I just classified the configures that I know and user will change during their deploy. If you find some configures I forget, please let me know.

**Comment by zhongjiajie** [comment]
I will submit PR for our website to flush how we use `install.sh` later

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Review by lenboo — APPROVED** [review]
+1

**Comment by zhongjiajie** [comment]
Thanks

**Pull request merged**